pkg/console/tlsconfig/tlsconfig.go

```diff
@@ -120,10 +118,21 @@ func (w *watch) reloadTlsProfile() {
 	{
 		// TODO: only compute human readable strings on debug level.
 		// For now, there is no easy way to test for logging level.
-		cipherNames := crypto.CipherSuitesToNamesOrDie(ciphers)
-		minVersionName := crypto.TLSVersionToNameOrDie(minVersion)
-		log.Log.V(1).Infof("Set min TLS version: %s", minVersionName)
-		log.Log.V(1).Infof("Set ciphers: %s", strings.Join(cipherNames, ", "))
+		if minVersion == 0 {
+			log.Log.V(1).Infof("Min TLS version was not set in the config file. Using default.")
+		} else {
+			log.Log.V(1).Infof("Set min TLS version: %s", tls.VersionName(minVersion))
+		}
+
+		if ciphers == nil {
+			log.Log.V(1).Infof("Ciphers were not set in the config file. Using default.")
+		} else {
+			cipherNames := make([]string, 0, len(ciphers))
+			for _, cipher := range ciphers {
+				cipherNames = append(cipherNames, tls.CipherSuiteName(cipher))
+			}
+			log.Log.V(1).Infof("Set ciphers: %s", strings.Join(cipherNames, ", "))
+		}
 	}
```

Comment:
what is the reason for confining this code in a `{}` code block?

Reply:
Previously it was to limit scope of temporary variables. It was supposed to be only for logging.
Now with the if statements, there is no reason.
If you want, I can remove them.

Reply:
I see. Your call! :blush:

Reply:
Then let's keep it as is. I don't want to waste resources running the CI again because of a cosmetic change.